ui/pl.json

```diff
@@ -0,0 +1,24 @@
+{
+  "building-delete": "Usuń",
+  "building-move": "Przesuń",
+  "contact_completed": "Kontrakt został wypełniony.",
+  "contact_unlocked": "Nowy kontrakt został odblokowany.",
+  "equipment_unknown": "Błąd: Nieznany ekwipunek.", //Review and context needed here!
+  "equipment_type_unknown": "Błąd: Nieznany typ ekwipunku.", //Review and context needed here!
+  "equipment_space_already_occupied": "Błąd: To miejsce zostało już zajęte.", //Review and context needed here!
+  "equipment_missing_resources": "Błąd: Za mała ilość zasobów aby wyprodukować ten rodzaj ekwipunku.", //Review and context needed here!
+  "market_exchange_no_enough_money": "Błąd: Za mało pieniędzy na tę transakcję.",
+  "market_exchange_no_enough_resource": "Błąd: Za mało zasobów na tę transakcję.", //Review and context needed here!
+  "market_exchange_no_enough_space": "Błąd: Za mało miejsca na tę transakcję.",
+  "popup-close": "Zamknij",
+  "loading-game": "Ładowanie gry...",
+  "money-format": "$%amount%",
+  "resource-buy": "Kup ($%price%)",
+  "resource-sell": "Sprzedaj ($%price%)",
+  "work-in-progress": "Work in progress", //I think we can leave this unchanged. I guess we could add "Prace w toku" or "W trakcie tworzenia". Up to you
+  "delete-building-message": "Czy chciałbyś usunąć ten budynek?", //Can we specify what "building" means in this context?
```

Comment:
Actually, it shouldn't be "building", if we keep the "equipment" word from everywhere

Reply:
This is where it gets tricky. In this case using word equipment in Polish wouldn't make much sense and it sounds odd

Reply:
We can use the same word as decided below :)